lib/msal-browser/docs/caching.md

```diff
@@ -1,3 +1,70 @@
+
+## Cache storage
+
+You can configure the cache storage location via the configuration object that is used to instantiate MSAL:
+
+```typescript
+import { PublicClientApplication, BrowserCacheLocation } from "@azure/msal-browser";
+
+const pca = new PublicClientApplication({
+    auth: {
+        clientId: "Enter_the_Application_Id_Here", // e.g. "b1b60dca-c49d-496e-9851-xxxxxxxxxxxx" (guid)
+        authority: "https://login.microsoftonline.com/Enter_the_Tenant_Info_Here", // e.g. "common" or your tenantId (guid),
+        redirectUri: "/"
+    },
+    cache: {
+       cacheLocation: BrowserCacheLocation.SessionStorage // "sessionStorage"
+    }
+});
+```
+
+By default, MSAL stores the various authentication artifacts it obtains from the IdP in browser storage using the [Web Storage API](https://developer.mozilla.org/docs/Web/API/Web_Storage_API) supported by all modern browsers. Accordingly, MSAL offers two methods of persistent storage: `sessionStorage` (default) and `localStorage`. In addition, MSAL provides `memoryStorage` option which allows you to opt-out of storing the cache in browser storage.
+
+| Cache Location   | Cleared on              | Shared between windows/tabs | Redirect flow supported |
+|------------------|-------------------------|-----------------------------|-------------------------|
+| `sessionStorage` | window/tab close        | No                          | Yes                     |
+| `localStorage`   | user logout             | Yes                         | Yes                     |
+| `memoryStorage`  | page refresh/navigation | No                          | No                      |
+
+> :bulb: While the authentication state may be lost in session and memory storage due to window/tab close or page refresh/navigation, respectively, users will still have an active session with the IdP as long as the session cookie is not expired and might be able to re-authenticate without any prompts.
+
+### Cookie storage
+
+MSAL Browser can be configured to use cookies for storing temporary authentication artifacts. This is useful for apps that need to support older browsers such as Internet Explorer. Note that when this option is chosen, tokens themselves are still stored in browser or memory storage. Please refer to [configuration](./configuration.md#cache-config-options) for more.
+
+### Security
+
+We consider session/local storage secure as long as your application does not have cross-site scripting (XSS) and related vulnurabilities. Please refer to the [OWASP XSS Prevention Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html) for securing your applications against XSS. If you are still concerned, we recommend using the `memoryStorage` option instead.
```

Comment:
Do we need to flag the OWASP page as being an external link?

Reply:
Hmm, I would say orgs like OWASP and MDN need no special mentioning, but if we were to flag these, what would that look like? :/